blueprints/app/files/.eslintrc.js

```diff
@@ -52,12 +52,11 @@ module.exports = {
         browser: false,
         node: true,
       },
-      plugins: ['node'],
-      extends: ['plugin:node/recommended'],<% if (blueprint === 'app') {%>
+      extends: ['plugin:n/recommended'],<% if (blueprint === 'app') {%>
       rules: {
-        // this can be removed once the following is fixed
-        // https://github.com/mysticatea/eslint-plugin-node/issues/77
-        'node/no-unpublished-require': 'off',
+        // this can be removed once the following PR is merged
+        // https://github.com/eslint-community/eslint-plugin-n/pull/57
```

Comment:
The PR was merged and [released](https://github.com/eslint-community/eslint-plugin-n/releases/tag/15.4.0) so we can simply remove this rule now.